sdk/core/azure-core/azure/core/pipeline/policies/_base.py

```diff
@@ -61,11 +61,8 @@ class HTTPPolicy(abc.ABC, Generic[HTTPRequestTypeVar, HTTPResponseTypeVar]):
     :type next: ~azure.core.pipeline.policies.HTTPPolicy or ~azure.core.pipeline.transport.HttpTransport
```

Comment:
And we also want to update this line, right?

Reply:
Actually, the whole doc is wrong, `next` is not an init parameter. Removing the whole thing.